CSV_filter.java

```diff
@@ -0,0 +1,44 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CSV_filter {
+
+    static Map<String, String> map = new HashMap<String, String>();
+
+    public static void main(String[] args) throws IOException {
+
+        Path DataDir = Path.of("src", "main","resources","Taxidatensatz.csv");
```

Comment:
Variable names  should start with a lowercase letter
```suggestion
        Path dataDir = Path.of("src", "main","resources","Taxidatensatz.csv");
```

Reply:
Cool!